src/main/java/reposense/git/GitLog.java

```diff
@@ -10,10 +10,12 @@
 
 /**
  * Contains git log related functionalities.
- * Git log is responsible to obtain the commit logs and the authors' info.
```

Comment:
Originally is because this line seems to belong to the `get` method instead, now that there are 2 methods in this class, it may seem inappropriate to put it there. 
Will put it back.